src/bp/core/services/telemetry/legacy-stats.ts

```diff
@@ -0,0 +1,186 @@
+import LicensingService from 'common/licensing-service'
+import { machineUUID } from 'common/stats'
+import { ConfigProvider } from 'core/config/config-loader'
+import Database from 'core/database'
+import { UserRepository } from 'core/repositories'
+import { TelemetryRepository } from 'core/repositories/telemetry_payload'
+import { TYPES } from 'core/types'
+import { inject, injectable } from 'inversify'
+import ms from 'ms'
+import os from 'os'
+import uuid from 'uuid'
+import yn from 'yn'
+
+import { GhostService } from '..'
+import AuthService from '../auth/auth-service'
+import { BotService } from '../bot-service'
+import { CMSService } from '../cms'
+import { JobService } from '../job-service'
+import { WorkspaceService } from '../workspace-service'
+
+import { TelemetryStats } from './telemetry-stats'
+
+@injectable()
+export class LegacyStats extends TelemetryStats {
+  protected url: string
+  protected lock: string
+
+  constructor(
+    @inject(TYPES.GhostService) ghostService: GhostService,
+    @inject(TYPES.Database) database: Database,
+    @inject(TYPES.LicensingService) licenseService: LicensingService,
+    @inject(TYPES.JobService) jobService: JobService,
+    @inject(TYPES.TelemetryRepository) telemetryRepo: TelemetryRepository,
+    @inject(TYPES.ConfigProvider) private config: ConfigProvider,
+    @inject(TYPES.CMSService) private cmsService: CMSService,
+    @inject(TYPES.AuthService) private authService: AuthService,
+    @inject(TYPES.UserRepository) private userRepository: UserRepository,
+    @inject(TYPES.BotService) private botService: BotService,
+    @inject(TYPES.WorkspaceService) private workspaceService: WorkspaceService
+  ) {
+    super(ghostService, database, licenseService, jobService, telemetryRepo)
+    this.url = 'https://telemetry.botpress.io/ingest'
+    this.lock = 'botpress:telemetry-legacyStats'
+  }
+
+  protected async getStats() {
+    const config = await this.config.getBotpressConfig()
+
+    return {
+      schema: '1.0.0',
+      timestamp: new Date().toISOString(),
+      uuid: uuid.v4(),
+      server: await this.getServerStats(),
```

Comment:
Just being picky here, can you also use getSchema like you do in actions.ts? Keys would have the same order :)

Reply:
This is legacy code and it is actually not the same schema as our other stats ever though is looks very similar. This should get phased out by the end of the summer.